.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Bug Report
+about: Report a problem with qBittorrent to help us resolve it.
+title: ''
+labels: ''
+assignees: ''
+---
+
+# Bug report
+
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED
+  AS INCOMPLETE/INVALID
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following requirements below,
+put an "x" (without the quotes) between the square brackets.
+
+When doing so, ensure that no space is left between the square brackets and the "x", like so: [x]
+-->
+
+- [ ] I have read the **issue reporting guidelines** in the [contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md), so I know how to submit a good bug report with the required information
+- [ ] I have verified that **the issue is not fixed and is reproducible** in the **[latest version](https://www.qbittorrent.org/download.php)**
+- [ ] I have **searched** the issue tracker for similar bug reports including closed ones **to avoid posting a duplicate**
+- [ ] This is **not a support issue or question**, or the [wiki](https://github.com/qbittorrent/qBittorrent/wiki) did not contain the required information and the [forum](https://qbforums.shiki.hu/) and [subreddit](https://www.reddit.com/r/qBittorrent/) were not able to assist me, so I am asking here
+- [ ] I have **pasted/attached the settings file and relevant log(s)** (at least the most recent one) in the **Attachments** section at the bottom of the report and filled in all the other required information below
+
+## Description
+
+<!--
+################################## IMPORTANT ##################################
+
+Delete each "(type here)" indicator and type your text in their place in the subsections below.
+
+Please make sure the description is worded well enough to be understood.
+Provide steps to reproduce the issue, any additional relevant information, suggested solution (if applicable) and as much context and examples as possible.
+For more information consult the Contributing Guidelines at https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md.
+
+Do not forget about the mandatory attachments!
+
+Use the Preview tab before posting to make sure your report looks like it is formatted properly.
+-->
+
+### qBittorrent version, install method (installer, compiled from git, ...), and operating system version
```

Comment:
>and operating system **version**

Has there ever been an issue that was verified to be caused by a specific OS version? Unless you mean win 7 8 10 in which case "version" is redundant because it's always mentioned.

 >install method (installer, compiled from git, ...)

I would change that to something that would make the reporter mention it IF it's anything not standard. I wouldn't want to read in 95% of issues "Installed from the official site".

Reply:
> Has there ever been an issue that was verified to be caused by a specific OS version?

Maybe not 100% by us directly **BUT** Builds 1903/1909 are notorious for HIGH CPU Usage & HIGH DPC Latency issues.

Also there's been issues with VPN's up to build 1909.

**Version** - would possibly be insignificant if Minimum OS Build is raised.....(Qt6 etc.)

Reply:
resolved.